src/conn/api_calc_modify.c

```diff
@@ -105,12 +105,12 @@ __cm_fingerprint(const uint8_t *p)
 }
 
 /*
- * wiredtiger_calc_modify --
+ * __wt_calc_modify --
  *     Calculate a set of WT_MODIFY operations to represent an update.
  */
 int
-wiredtiger_calc_modify(WT_SESSION *wt_session, const WT_ITEM *oldv, const WT_ITEM *newv,
-  size_t maxdiff, WT_MODIFY *entries, int *nentriesp)
+__wt_calc_modify(WT_SESSION *wt_session, const WT_ITEM *oldv, const WT_ITEM *newv, size_t maxdiff,
```

Comment:
I'd change this to take a `WT_SESSION_IMPL *session` as the argument, since it's an internal routine, and then its internal callers won't need to cast anything.

Reply:
Changed as suggested.